benchmarks/simulator.py

```diff
@@ -0,0 +1,37 @@
+from pysb.examples import earm_1_0
+from pysb.simulator import ScipyOdeSimulator, CupSodaSimulator
+import numpy as np
+import timeit
+
+
+class Earm10ODESuite(object):
+    def setup(self):
+        self.nsims = 100
+        self.timer = timeit.default_timer
+        self.model = earm_1_0.model
+        self.parameter_set = np.ones((self.nsims, len(self.model.parameters)))
```

Comment:
The initialization to all ones followed by updates with `*=` seems really roundabout, unless I'm missing something obvious. The following seems clearer if the goal is just to expand the parameter values list n times:
```python
self.parameter_set = np.repeat([[p.value for p in self.model.parameters]], self.nsims, axis=0)
```